fabtests/unit/getinfo_test.c

```diff
@@ -1032,10 +1032,10 @@ int main(int argc, char **argv)
 			ft_parseinfo(op, optarg, hints, &opts);
 			break;
 		case 'h':
-			usage();
+			usage(argv[0]);
 			return EXIT_SUCCESS;
```

Comment:
for some reason getinfo_test.c is different from others where -h returns EXIT_SUCCESS while others return EXIT_FAILURE

Reply:
I think it is safe to make this consistent with others.

Reply:
done